src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -141,16 +141,24 @@ public String parser(String userInput, ArrayList<Project> projectList) {
                 }
                 break;
             case SPRINT:
-
+                Hashtable<String, String> newParams = new Hashtable<>();//to be removed after standardisation
                 switch (action.toLowerCase()) {
                 case CREATE:
                     new CreateSprintCommand(parameters, projectList).execute();
                     break;
                 case ADDTASK:
-                    new AddSprintTaskCommand(params, projectList).execute();
+                    //Convert Arraylist to Hashtable to simulate standardisation
+                    for (int i = 0; i < params.size(); i++) {
```

Comment:
Sorry, missed this out, this can be SLAP-ed right? So that `REMOVETASK` can use the same function.

Reply:
This is just a temporary change before @homingjun make a permanent change to parser.

Reply:
ArrayList should be totally removed from parser.

Reply:
Alright